geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -339,6 +342,7 @@ impl SlotStatus {
             SlotStatus::FirstShredReceived => "first_shread_received",
             SlotStatus::Completed => "completed",
             SlotStatus::CreatedBank => "created_bank",
+            SlotStatus::Dead(_error) => "dead",
```

Comment:
should we include the error in the string here?

Reply:
The interface here returns &'static str -- so it is not possible to include the the String field in Dead status. The plugin can choose to  customize the deserialization the error into another field.